matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/mapper/RoomSummaryMapper.kt

```diff
@@ -43,6 +43,7 @@ internal class RoomSummaryMapper @Inject constructor(private val timelineEventMa
                 topic = roomSummaryEntity.topic ?: "",
                 avatarUrl = roomSummaryEntity.avatarUrl ?: "",
                 isDirect = roomSummaryEntity.isDirect,
+                directUserId = roomSummaryEntity.directUserId ?: "",
```

Comment:
no need for `?: ""` since `RoomSummary.directUserId` can be null

Reply:
I fixed it